content/questions/actions/question-079.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 079"
```

Comment:
Can you also change the filename ? Sadly this is the way it's currently done in this project that there are conflicts if someone merges questions before other PR is merged
```suggestion
title: "Question 093"
```